devops/ansible/roles/large_image/molecule/mapnik/tests/request_tile_in_container.py

```diff
@@ -0,0 +1,20 @@
+import large_image
+import urllib
+import pytest
+
+
+@pytest.mark.parametrize("test_url, output", [
+    ('https://download.osgeo.org/geotiff/samples/spot/chicago/SP27GTIF.TIF',
+     '/tmp/chicago.tif')
+])
+def test_mapnik_tile_source(test_url, output):
```

Comment:
I hope we can move these tests up to the main `large_image` package in the future. It's a shame that that these nice, compact tests are only usable in the context of Molecule testing. In the future, hopefully the Molecule tests can just run these selectively, taking them from the whole set of normal `large_image` tests.